src/main/java/uk/gov/companieshouse/disqualifiedofficersdataapi/controller/HealthCheckController.java

```diff
@@ -1,15 +1,14 @@
 package uk.gov.companieshouse.disqualifiedofficersdataapi.controller;
 
-import org.springframework.http.HttpStatus;
 import org.springframework.http.ResponseEntity;
 import org.springframework.web.bind.annotation.GetMapping;
 import org.springframework.web.bind.annotation.RestController;
 
 @RestController
 public class HealthCheckController {
 
-    @GetMapping("/healthcheck")
-    public ResponseEntity<Void> healthcheck() {
-        return ResponseEntity.status(HttpStatus.OK).build();
+    @GetMapping("disqualified-officers/healthcheck")
+    public ResponseEntity<String> healthcheck() {
+        return ResponseEntity.ok().body("OK");
```

Comment:
Could we use spring actuator rather have our own healthcheck?

Reply:
I based this healthcheck off of the previous few PR's that we implemented a health check, so I was trying to keep it standard with those. I looked for some examples where actuator was being used already and I couldn't find any in the repos I checked. Is this a necessary change? If so we will need to be doing this for all the apps we upgrade not just some.

Reply:
I think so yeah, it is our agreed healthcheck library going forwards.

https://github.com/companieshouse/strike-off-objections-api/pull/148/files

Reply:
If it could be backported to other apis that would be good and maybe included as part of a ticket acceptance criteria

Reply:
Done, followed example shared